### ISSUE TITLE
Its not using the .jshintrc in project root

### ISSUE THREAD

**Issue by crucialfelix** [AtomLinter/linter-jshint#177]
It seems that its ignoring the .jshintrc in the project folder

`W119 - 'export' is only available in ES6 (use esnext option)`

but I do have that set:

      // .jshintrc
      "esnext": true,

and linting on the command line is fine (same jshint 2.8):

```shell
supercolliderjs ❯ npm run lint

> supercolliderjs@0.8.2 lint /Users/crucial/supercolliderjs
> jshint src/*.es6 bin/*.js examples/*.js
```

Maybe you should pass --config to jshint rather than assuming that it will find it by itself. 

What path does jshint think it is at ?

https://github.com/AtomLinter/linter-jshint/blob/master/lib%2Fmain.coffee#L57

You pass in the text and the --filename flag but I'm not sure that jshint knows to search for a .jshintrc given that filename.

**Comment by Arcanemagus** [comment]
Looks like either the cwd needs to be set (probably the better solution) or the `--config` option needs to be specified.

**Comment by crucialfelix** [comment]
It looks like options is passed all the way through to `ChildProcess` so you could supply `cwd`

https://github.com/AtomLinter/atom-linter/blob/master/lib%2Fhelpers.coffee
https://github.com/atom/atom/blob/8fb4e887df800438b2deeb23a2c23004b9ff358a/src%2Fbuffered-process.coffee#L194
https://nodejs.org/api/child_process.html#child_process_asynchronous_process_creation

**Comment by brody182** [comment]
how do I use ES6?

**Comment by crucialfelix** [comment]
Use this: https://atom.io/packages/linter-jscs
and/or this: https://atom.io/packages/linter-eslint

I have linter-jshint disabled now and the above two work great for es5 and es6 code.
